src/Course/ExactlyOne.hs

```diff
@@ -323,15 +344,19 @@ showAndTell t0 =
 --   a) What is the signature of `show`? (Hint, use `:type show` instead of `:info show`.)
 --   b) In our own words, what different kinds of info does `:info Show` give us?
 question_ExactlyOne_5 :: (P.String, P.String)
-question_ExactlyOne_5 = error "todo"
+question_ExactlyOne_5 =
+  ( "`show` is a generic function with constrained type argument `Show a` that takes a value of type `a` and returns a string. I'm guessing this is the function that produces the formatted output for a value."
```

Comment:
> constrained type argument `Show a`

It's more like we `show` has a constrained type argument `a`. the constraint on `a` happens to be `Show a`.